File: 2037-minimum-number-of-moves-to-seat-everyone/2037-minimum-number-of-moves-to-seat-everyone.c


int minMovesToSeat(int* seats, int seatsSize, int* students, int studentsSize){
    
    
    int temp = 0;
    int sum =0;
    for(int i = 0;i<seatsSize-1;i++){       
        for(int j=0;j<seatsSize-i-1;j++){          
            if(seats[j]>seats[j+1]){               
                temp = seats[j];
                seats[j] = seats[j + 1];
                seats[j + 1] = temp;
                }
            if(students[j] > students[j+1]){
                temp = students[j];
                students[j] = students[j+1];
                students[j+1] = temp;
                }
            }
        }
    for(int i = 0; i < seatsSize;i++){
        if((seats[i] - students[i]) >= 0){
            sum += (seats[i] - students[i]);
        }
        else{
            sum += (students[i] - seats[i]);
        }
    }
    return sum;
}

File: 2037-minimum-number-of-moves-to-seat-everyone/2037-minimum-number-of-moves-to-seat-everyone.java
class Solution {
    public int minMovesToSeat(int[] seats, int[] students) {
        Arrays.sort(seats);
        Arrays.sort(students);
        int total = 0;
        for (int i = 0; i < seats.length;i++){
            total += Math.abs(students[i]-seats[i]);
        }
        return total;   
    }
}